app/javascript/components/student/Editor.tsx

```diff
@@ -269,6 +279,18 @@ export function Editor({
     return abort
   }, [abort])
 
+  useEffect(() => {
+    if (!submission) {
+      return
+    }
+
+    fetchJSON(submission.links.testRun, {
```

Comment:
Do we need to add the abort code used in https://github.com/exercism/v3-website/pull/249#pullrequestreview-524090289 here too?

Reply:
Yes. I've added this now :)